packages/@expo/cli/src/utils/errors.ts

```diff
@@ -57,7 +57,7 @@ export function logCmdError(error: any): never {
   }
   if (error instanceof AbortCommandError || error instanceof SilentError) {
     // Do nothing, this is used for prompts or other cases that were custom logged.
-    process.exit(0);
+    process.exit(1);
```

Comment:
Seems like it's worth a try. We should just keep an eye out for undesired output for things like ctrl+c errors.